services/core/methods/controllers/transactions.js

```diff
@@ -111,6 +111,7 @@ const getTransactionsStatistics = async ({
 		dateFormat,
 		dateTo,
 		dateFrom,
+		limit: 10000, // Supress the default value coming from MySQL layer
```

Comment:
Since `limit` is concerning the DB layer, I would prefer keeping the controller clean and rather prefer to move it to the `getSelector` method in `services/core/shared/core/transactionStatistics.js`.